pkg/resources/schema_grant.go

```diff
@@ -16,6 +16,7 @@ import (
 
 var validSchemaPrivileges = NewPrivilegeSet(
 	privilegeAddSearchOptimization,
+	privilegeCreateDynamicTable,
```

Comment:
For completeness, it should also be added to the `pkg/sdk/privileges.go` file (`SchemaPrivilege` type).